src/models/Event.js

```diff
@@ -39,27 +40,38 @@ const eventSchema = new Schema({
                     validator: (value) => validTime.test(value),
                     message: 'Invalid time format! Example: hh:mm (24h)'
                 }
-                
             }
-        }, { // Additional date and time (if needed)
-            date: { type: Date },
-            startTime: { type: String },
-            endTime: { type: String }
         }]
     },
-    imageUrl: { type: String, required: true },
+    imageUrl: {
+        type: String,
+        required: true,
+        validate: {
+            validator: (value) => validString.test(value),
+            message: "Invalid URL, must start with HTTP:///HTTPS://",
```

Comment:
Won’t the organisation be able to upload an image, not only passing an url for it?

Reply:
Did you mean to store "img" files locally?